InvenTree/build/tasks.py

```diff
@@ -98,3 +102,62 @@ def check_build_stock(build: build.models.Build):
         recipients = emails.values_list('email', flat=True)
 
         InvenTree.tasks.send_email(subject, '', recipients, html_message=html_message)
+
+
+def notify_overdue_build_order(bo: build.models.Build):
+    """Notify appropriate users that a Build has just become 'overdue'"""
+
+    targets = []
+
+    if bo.issued_by:
+        targets.append(bo.issued_by)
+
+    if bo.responsible:
+        targets.append(bo.responsible)
+
+    name = _('Overdue Build Order')
+
+    context = {
+        'order': bo,
+        'name': name,
+        'message': _(f"Build order {bo} is now overdue"),
+        'link': InvenTree.helpers.construct_absolute_url(
+            bo.get_absolute_url(),
+        ),
+        'template': {
+            'html': 'email/overdue_build_order.html',
+            'subject': f'[Inventree] {name}',
```

Comment:
@SchrodingersGat could you reference the instance name here? Fir whitelabeling purposes

Reply:
I have refactored these new notifications (and all the existing ones) to reference the instance name when sending emails.